client/src/components/SideNav.jsx

```diff
@@ -4,7 +4,7 @@ import { useNavigate } from 'react-router-dom';
 const navLinks =[
   { id: 1, title: 'Flags Dashboard', path: "/" },
   { id: 2, title: 'Metrics', path: '/metrics' },
-  { id: 3, title: 'Get an SDK Key', path: "/"}
+  { id: 3, title: 'Get an SDK Key', path: "/sdkKey"}
```

Comment:
Maybe we could rename this to just "SDK Key"? Since it can be used for creating or viewing the existing one